config/sevntu_suppressions.xml

```diff
@@ -31,4 +31,8 @@
     and do not have coverage problems due to imports from java.util.stream. -->
     <suppress id="ForbidInterfacesImportFromJavaUtilStream"
               files=".*[\\/]src[\\/](test|it)[\\/]"/>
+
+    <!-- Tone down the checking for test code -->
+    <suppress checks="ForbidCertainImports"
+              files="DetailASTTest\.java"/>
```

Comment:
@romani This configuration item is what gave me the error: https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_sevntu_checks.xml#L120-L127
It is because I have to use a `AbstractCheck` to force Java grammar to run on file.

There is no way to modify check as we can't identify what folder we are in using check, or identify what file we are in.
Besides suppression, the only other way is to change the test to not use the full `verify`.